Gemfile

```diff
@@ -2,3 +2,11 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in kafo_parsers.gemspec
 gemspec
+
+puppet_version = ENV['PUPPET_VERSION'] || ENV['puppet']
+puppet_spec = puppet_version ? "~> #{puppet_version}" : '< 5.0.0'
+gem 'puppet', puppet_spec
+
+if puppet_version.nil? || puppet_version >= '3.7'
```

Comment:
Would it be possible to rather load custom Gemfile.local.rb which would be created in jenkins job? It seems a bit safer, `puppet` env variable looks too generic. If this is just for jenkins purposes, why do we need two env variables?